src/Cpanel/PluginActions.php

```diff
@@ -21,4 +51,51 @@ public function applyDefaultSettings()
     {
         // Do nothing
     }
+
+    public function getConfig()
+    {
+        $this->getUserConfig();
+        $this->getComposerJson();
+
+        //Clone the config to manipulate
+        $config = array_merge(array(), self::$CONFIG);
```

Comment:
I think you would only need to do this if you weren't merging into an empty `array()`.  None of the child arrays will conflict since they don't exist on the empty `array()`